packages/catala-format/catala-format.0.1.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+maintainer: "vincent.botbol@inria.fr"
+authors: [ "Vincent Botbol" ]
+
+homepage: "https://github.com/CatalaLang/catala-format"
+bug-reports: "https://github.com/CatalaLang/catala-format"
+dev-repo: "git+https://github.com/CatalaLang/catala-format.git"
+
+license: "Apache-2.0"
+depends: [
+  "topiary" {>= "0.3.0"}
+]
```

Comment:
It might be interesting for you to create a `conf-topiary` package and then depend on either `topiary` or `conf-topiary`. This would allow users to install `catala-format` using an installation of `topiary` that's not managed by `opam`.

It's more of a suggestion for later. And even then it's only if you think it'd be nice. But just thought I would mention.